app/Repositories/ModularPageRepository.php

```diff
@@ -287,4 +276,78 @@ public function organizePromoItemsByOption(array $data)
 
         return $data;
     }
+
+    /**
+    * {@inheritdoc}
+    */
+    public function getHeroFromComponents(array $promos, $data)
+    {
+        // Set hero from components
+        $hero = collect($promos['components'])->reject(function ($data, $component_name) {
+            return !str_contains($component_name, 'hero');
+        })->toArray();
+
+        if (!empty($hero)) {
+            $hero_key = array_key_first($hero);
+
+            // Minimal hero overwrites any other hero data
+            if ($hero_key === 'misb-minimal-hero' && !empty($promos['components'][$hero_key]['component']['filename'])) {
```

Comment:
Perhaps this isn't needed in this project? 
'misb-minimal-hero'

Reply:
Ty! I'm sorry I missed that